src/components/shared/Dropdown.js

```diff
@@ -58,6 +60,7 @@ export class Dropdown extends Component<Props, State> {
         className="dropdown-mask"
         onClick={this.toggleDropdown}
         style={{ display: this.state.dropdownShown ? "block" : "none" }}
+        role="presentation"
```

Comment:
it has a click handler so it does not seem to be for "presentation" purposes only.